lib/run/options.js

```diff
@@ -57,35 +57,41 @@ var _ = require('lodash'),
      */
     configLoaders = {
         collection: function (value, callback) {
-            var done = function (err, collection) {
-                if (err) {
-                    return callback(err);
-                }
+            var V2_COLLECTION_PROPERTY = 'info.schema',
+                COLLECTION_TRANSFORMER_OPTION = { inputVersion: '1.0.0', outputVersion: '2.0.0' },
 
-                // ensure that the collection option is present before starting a run
-                if (!_.isObject(collection)) {
-                    return callback(new Error('newman: collection could not be loaded'));
-                }
+                done = function (err, collection) {
+                    if (err) {
+                        return callback(err);
+                    }
 
-                // ensure that the collection reference is an SDK instance
-                // @todo - should this be handled by config loaders?
-                collection = new Collection(Collection.isCollection(collection) ?
-                    // if the option contain an instance of collection, we simply clone it for future use
-                    // create a collection in case it is not one. user can send v2 JSON as a source and that will be
-                    // converted to a collection
-                    collection.toJSON() : collection);
+                    // ensure that the collection option is present before starting a run
+                    if (!_.isObject(collection)) {
+                        return callback(new Error('newman: collection could not be loaded'));
+                    }
 
-                callback(null, collection);
-            };
+                    // ensure that the collection reference is an SDK instance
+                    // @todo - should this be handled by config loaders?
+                    collection = new Collection(Collection.isCollection(collection) ?
+                        // if the option contain an instance of collection, we simply clone it for future use
+                        // create a collection in case it is not one. user can send v2 JSON as a source and that will be
+                        // converted to a collection
+                        collection.toJSON() : collection);
 
-            if (!_.isString(value)) { return done(null, value); }
+                    callback(null, collection);
+                };
+
+            if (_.isObject(value)) {
```

Comment:
I think this detection should explicitly check for v1 collection by looking for (requests array or folders array). Otherwise there will be a lot of false positives.